larpandoracontent/LArControlFlow/StitchingCosmicRayMergingTool.cc

```diff
@@ -537,11 +537,14 @@ void StitchingCosmicRayMergingTool::StitchPfos(const MasterAlgorithm *const pAlg
         for (const ParticleFlowObject *const pPfo : pfoList) pfoVector.push_back(pPfo);
         std::sort(pfoVector.begin(), pfoVector.end(), LArPfoHelper::SortByNHits);
 
+        PfoVector reducedPfoVector;
+        const ParticleFlowObject *const selectedPfoToEnlarge(this->ReduceToLongestStitch(pfoVector, reducedPfoVector, pPfoToEnlarge, pfoToLArTPCMap));
+
```

Comment:
Pandora style guide is all pointer variables start with a small p, e.g. `pSelectedPfoToEnlarge`.

Also, style guide is for const function arguments first in argument list, followed by non-const arguments, e.g. containers that you will fill. `pfoVector` can be passed by const reference here, with only non-const argument being `reducedPfoVector` that you will fill.

Reply:
Addressed. The reason why pfoVector was not passed by const reference was because it could not be passed as const to SelectLongestStitch called inside ReduceToLongestStitch, as that one would resize pfoVector to return it if the rest of the logic failed. I agree it is better to have it as constant so I changed the final bit of SelectLongestStitch accordingly.